app/services/direction_service.rb

```diff
@@ -0,0 +1,12 @@
+class DirectionService
+  def self.conn
+    Faraday.new(url: "http://www.mapquestapi.com") do |faraday|
+      faraday.params['key'] = ENV['mapquest_api']
+    end
+  end
+
+  def self.get_travel_info(starting_location, ending_location)
+    response = conn.get("/directions/v2/route?from=#{starting_location}&to=#{ending_location}")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+end
```

Comment:
This is me being super picky -- One thing you could refactor to better align with rails convention is using some sort of block notation to pass in parameters rather than adding them in the string of the url. You're doing this in your conn method, but then you don't do it in the request on line 9. Again, super picky, but jsut something to consider ;)